includes/results.inc

```diff
@@ -697,7 +697,8 @@ class IslandoraSolrFacets {
   public static function init($islandora_solr_query) {
     self::$islandoraSolrQuery = $islandora_solr_query;
     self::$facet_fields = isset($islandora_solr_query->islandoraSolrResult) ? $islandora_solr_query->islandoraSolrResult['facet_counts']['facet_fields'] : array();
-    self::$facet_dates = isset($islandora_solr_query->islandoraSolrResult) ? $islandora_solr_query->islandoraSolrResult['facet_counts']['facet_dates'] : array();
+    // XXX: isset() checking, as newer Solrs (7) won't return a value.
+    self::$facet_dates = isset($islandora_solr_query->islandoraSolrResult['facet_counts']['facet_dates']) ? $islandora_solr_query->islandoraSolrResult['facet_counts']['facet_dates'] : array();
```

Comment:
Same as previous review, maybe we can just use ranges

Reply:
Same answer as above, but if people want me to try removing the date stuff I can certainly do that. Not sure if that should be part of this or separate though?